### DISCUSSION TITLE
Icon request: fa-fireworks

### DISCUSSION THREAD

**Discussion by bigfoot90** [FortAwesome/Font-Awesome#21185] [Icon Requests]
Add `fa-fireworks` icon for celebrating events like "**New Year**"

Some examples:
![image](https://user-images.githubusercontent.com/4598274/32601201-29a8121c-c542-11e7-95e6-b2e2cf231a0f.png)
![image](https://user-images.githubusercontent.com/4598274/32601207-2ea2d400-c542-11e7-8096-4192ef7e1517.png)
![image](https://user-images.githubusercontent.com/4598274/32601266-549ddf88-c542-11e7-8580-d64af19f31ab.png)
![image](https://user-images.githubusercontent.com/4598274/32601222-3a6969c0-c542-11e7-9395-dc306b27d422.png)
![image](https://user-images.githubusercontent.com/4598274/32601320-850ce5ba-c542-11e7-8f0b-fa970680f807.png)

**Comment by NityaCode** [comment]
Please add different firework icons.

**Comment by danielcongrove** [comment]
+1, a firework explosion would be good for 4th of July & New Years holidays

**Comment by hugoattal** [comment]
For those looking for a replacement, you can take a look at the `star-christmas` in the meantime. Not ideal, but can do the job!
https://fontawesome.com/icons/star-christmas?f=classic&s=regular

![image](https://github.com/FortAwesome/Font-Awesome/assets/4563971/5633228e-dc73-489d-afc7-f6fc87cdd2f0)


And for a free alternative, take a look ad the `bahai` logo (in solid)
https://fontawesome.com/icons/bahai?f=classic&s=solid

![image](https://github.com/FortAwesome/Font-Awesome/assets/4563971/7618ac11-c4c5-4669-968f-dd639c71e5e5)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
